plugin/trino-spooling-filesystem/src/main/java/io/trino/spooling/filesystem/FileSystemSpoolingManager.java

```diff
@@ -207,14 +210,8 @@ public SpooledSegmentHandle handle(SpooledLocation location)
         return new FileSystemSpooledSegmentHandle(encoding, queryId, uuid, Optional.of(encryptionHeadersTranslator.extractKey(location.headers())));
     }
 
-    private Location location(FileSystemSpooledSegmentHandle handle)
```

Comment:
can you keep the method so call spots are clearer (you will not have to do `fileSystemLayout.` in call spots.

Reply:
THe method will call the layout so it doesn't seem to make sense